doc/src/fix_langevin.txt

```diff
@@ -232,28 +236,37 @@ The keyword {gjf} can be used to run the "Gronbech-Jensen/Farago
 described in the papers cited below, the purpose of this method is to
 enable longer timesteps to be used (up to the numerical stability
 limit of the integrator), while still producing the correct Boltzmann
-distribution of atom positions.  It is implemented within LAMMPS, by
-changing how the random force is applied so that it is composed of
-the average of two random forces representing half-contributions from
-the previous and current time intervals.
-
-In common with all methods based on Verlet integration, the
-discretized velocities generated by this method in conjunction with
-velocity-Verlet time integration are not exactly conjugate to the
-positions.  As a result the temperature (computed from the discretized
-velocities) will be systematically lower than the target temperature,
-by a small amount which grows with the timestep.  Nonetheless, the
-distribution of atom positions will still be consistent with the
+distribution of atom positions.
+
+The current implementation provides the user with the option to output
+the velocity in one of two forms: {vfull} or {vhalf}, which replaces
+the outdated option {yes}. The {gjf} option {vfull} outputs the on-site
+velocity given in "Gronbech-Jensen/Farago"_#Gronbech-Jensen; this velocity
+is shown to be systematically lower than the target temperature by a small
+amount, which grows quadratically with the timestep.
+The {gjf} option {vhalf} outputs the 2GJ half-step velocity given in
+"Gronbech Jensen/Gronbech-Jensen"_#2Gronbech-Jensen; this velocity is shown
+to not have any linear statistical errors for any stable time step.
+An overview of statistically correct Boltzmann and Maxwell-Boltzmann
+sampling of true on-site and true half-step velocities is given in
+"Gronbech-Jensen_#1Gronbech-Jensen.
+Regardless of the choice of output velocity, the sampling of the configurational
+distribution of atom positions is the same, and linearly consistent with the
 target temperature.
 
-As an example of using the {gjf} keyword, for molecules containing C-H
-bonds, configurational properties generated with dt = 2.5 fs and tdamp
-= 100 fs are indistinguishable from dt = 0.5 fs.  Because the velocity
-distribution systematically decreases with increasing timestep, the
-method should not be used to generate properties that depend on the
-velocity distribution, such as the velocity auto-correlation function
-(VACF). In this example, the velocity distribution at dt = 2.5fs
-generates an average temperature of 220 K, instead of 300 K.
+An example of a reason why to use the {gjf} keyword is the freedom to take a larger time step,
```

Comment:
This whole block of text is redundant and/or beyond the scope of LAMMPS documentation. It should be deleted:

An example of a reason why to use the {gjf} keyword is the freedom to take a larger time step,
up to the stability limit, while maintaining robust statistics. It is crucial to
recall that while the equilibrium statistics is appropriately sampled, the correct dynamics
of the trajectories may not be for large time steps, as is the case for all thermostats.
All thermostats provide good statistics and dynamics for small time steps.
The 2GJ half-step velocity {vhalf} samples the correct velocity distribution for the {gjf} trajectory.

This updated implementation of the {gjf} thermostat includes the choice between
outputting either the on-site {vfull} or half-step {vhalf} velocity. The on-site
velocity has been updated to be the GJF on-site velocity, and the half-step
velocity is the statistically correct 2GJ velocity. The implementation
also takes advantage of Gaussian distributed random numbers in order to achieve
correct fluctuations.

Reply:
It has been removed.